modules/order/order.service.js

```diff
@@ -209,12 +211,19 @@ class OrdersService {
         itemsDiscount / exchangeRate;
     }
 
+    if (order.paymentMethod === PAYMENT_TYPES.CASH) {
```

Comment:
![image](https://user-images.githubusercontent.com/80056472/201737716-d63f5353-0896-4365-a9a8-1316d286caa6.png)
why we need two fields for order payment status in database? just leave it as boolean value and move this condition to admin part

Reply:
The payment status may change depending on the transaction when we paying by card. And the administrator cannot change isPaid when paymentMethod = CARD but it can change when customer paid by cash.
isPaid changing when admin click on checkbox but didn't change paymentStatus. Now it work correctly.